### ISSUE TITLE
 Error: Cypress had a problem

### ISSUE THREAD

**Issue by gamage1995** [applitools/netlify-plugin-visual-diff#13]
Netlify returns the following error when the visual diff plugin is invoked as an onPostBuild event

`The plugins file is missing or invalid.
Your 'pluginsFile' is set to '/opt/build/repo/.netlify/plugins/node_modules/netlify-plugin-visual-diff/visual-diff/plugins/index.js', but either the file is missing, it contains a syntax error, or threw an error when required. The 'pluginsFile' must be a '.js', '.ts', or '.coffee' file.
Or you might have renamed the extension of your 'pluginsFile'. If that's the case, restart the test runner.
Please fix this, or set 'pluginsFile' to 'false' if a plugins file is not necessary for your project.`

Error message
Error: Cypress had a problem
Could not find Cypress test run results
​
Plugin details
Package:        netlify-plugin-visual-diff
Version:        2.0.0
Repository:     git+https://github.com/jlengstorf/netlify-plugin-visual-diff.git
npm link:       https://www.npmjs.com/package/netlify-plugin-visual-diff
Report issues:  https://github.com/jlengstorf/netlify-plugin-visual-diff/issues

Error location
In "onPostBuild" event in "netlify-plugin-visual-diff" from Netlify app
      at onPostBuild (/.netlify/plugins/node_modules/netlify-plugin-visual-diff/index.js:84:16)

**Comment by colbyfayock** [comment]
hey @gamage1995 do you have your API key set as an environment variable in Netlify?

**Comment by gamage1995** [comment]
@colbyfayock Yes, I tried removing both the plugin and API key and adding again but the issue is still there.
